ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -263,7 +263,7 @@
  */
 
 @SuppressWarnings({"deprecation", "rawtypes"})
-public class Hive {
+public class Hive implements AutoCloseable {
```

Comment:
should it be Closeable or AutoCloseable?

Reply:
I used it in a try-with-resources block, hence it looked to be a good candidate for AutoCloseable